packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1622,7 +1623,7 @@ export class ContainerRuntime
 		);
 
 		let outerDeltaManager: IDeltaManager<ISequencedDocumentMessage, IDocumentMessage>;
-		const useDeltaManagerOpsProxy =
+		this.useDeltaManagerOpsProxy =
```

Comment:
probably worth setting this to be `false` by default rather than true?

Reply:
Updated